modules/ROOT/pages/ManualInstallDirections.adoc

```diff
@@ -69,3 +69,39 @@ But you can try to name your files so its name comes after the name of the depen
 If you need to know how to enable `developmentMode`,
 please refer to the xref:SMLConfiguration.adoc[SML Configuration Page].
 ====
+
+== Installing mods on Linux (Steam) as a player - SML version 2.2.1
+
+[WARNING]
+====
+_THIS IS UNSUPPORTED -> NO SUPPORT SHOULD BE EXPECTED FOR ANYONE ATTEMPTING THIS_.
+====
+
+. `cd <game root>/FactoryGame/Binaries/Win64`
+. Navigate to https://github.com/satisfactorymodding/SatisfactoryModBootstrapper/releases/tag/v2.0.11 and download into the `./Win64` folder:
+. . `msdia140.dll`
+. . `xinput1_3.dll`
+. `mkdir -p <game root>/loaders && cd <game root>/loaders`
+.  Navigate to https://github.com/satisfactorymodding/SatisfactoryModLoader/releases/tag/2.2.1 and download into the `./loaders` folder:
```

Comment:
Please change this to use a `latest` url (https://github.com/satisfactorymodding/SatisfactoryModLoader/releases/latest)

Reply:
Done! =)